niceman/distributions/conda.py

```diff
@@ -345,19 +363,18 @@ def identify_distributions(self, paths):
                 packages.append(package)
 
             # Give the distribution a name
-            if (len(conda_paths)) > 1:
-                env_name = 'conda_env-%d' % idx
+            # Determine name from path (Alt approach: use conda-env info)
+            if os.path.normpath(conda_path) == os.path.normpath(root_path):
+                env_name = "root"
```

Comment:
I suppose that one advantage of the alternative approach is that we wouldn't end up with the "root"/"base" mismatch for newer conda versions.